frontend/src/app/shared/components/table-pagination/table-pagination.component.html

```diff
@@ -32,17 +31,23 @@
         <li *ngFor="let pageNumber of pageNumbers"
             [ngClass]="{ 'op-pagination--item_current': pageNumber === pagination.page }"
             class="op-pagination--item">
-          <em *ngIf="pageNumber === pagination.page">{{ pageNumber }}</em>
+          <em
+            *ngIf="pageNumber === pagination.page"
+            [attr.aria-label]="text.pageNumberText(pageNumber)"
+            aria-current="page"
+            tabindex="0">
+            {{ pageNumber }}
+          </em>
           <button
             class="op-pagination--item-link"
             type="button"
             *ngIf="pageNumber !== pagination.page"
             (click)="showPage(pageNumber)"
             rel="next"
+            [attr.aria-label]="text.pageNumberText(pageNumber)"
             [textContent]="pageNumber"
           ></button>
         </li>
-
         <li *ngIf="postPageNumbers.length" class="op-pagination--space">…</li>
```

Comment:
There is a separator role that might be appropriate here: 

```suggestion
        <li *ngIf="postPageNumbers.length" class="op-pagination--space" role="separator">…</li>
```